src/components/Category.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import './Category.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { bookStatus } from '../redux/categories/categories';
 
-const Category = () => (
-  <div className="Category">
-    <div className="Category-text">
-      <button type="button">Check status</button>
-    </div>
-  </div>
-);
+const Category = () => {
+  const selectedCategory = useSelector((state) => state.categories);
+  const dispatchCategory = useDispatch();
 
+  const btnCheckStatus = () => {
+    dispatchCategory(bookStatus());
+  };
+  return (
+    <div className="Category">
+      <div className="Category-text">
+        <h2>{selectedCategory.Categories[0]}</h2>
+        <button onClick={btnCheckStatus} type="button">Check status</button>
+      </div>
+    </div>
+  );
```

Comment:
Note: This is only optional because it wasn't mentioned by the previous reviewer, but I highly recommend that you take it into consideration to improve your application.

[OPTIONAL] There's something wrong with the link to the `Categories` page, when I click on the navigation I get redirected to a blank page. Kindly make sure that you are routing the categories component correctly.

- It is important I can see the categories page and the button event displays `Under construction` which is dispatched from the redux store.